packages/sdks/mitosis.config.js

```diff
@@ -9,24 +9,40 @@ const getSeededId = () => {
 
 const isMitosisNode = (x) => x && x['@type'] === '@builder.io/mitosis/node';
 
+/**
+ * @type {import('@builder.io/mitosis'.MitosisConfig['options']['vue'])}
+ */
+const vueConfig = {
+  transpiler: { format: 'esm' },
+  namePrefix: (path) => (path.includes('/blocks/') ? 'builder' : undefined),
+  cssNamespace: getSeededId,
+  asyncComponentImports: true,
+};
+
 /**
  * @type {import('@builder.io/mitosis'.MitosisConfig)}
  */
 module.exports = {
   files: 'src/**',
-  targets: ['reactNative', 'vue', 'solid', 'svelte'],
-  transpiler: {
-    vue: { format: 'esm' },
-    svelte: { format: 'esm' },
-    solid: { format: 'esm' },
-    reactNative: { format: 'esm' },
-  },
+  targets: ['reactNative', 'vue2', 'vue3', 'solid', 'svelte'],
   options: {
-    vue: {
-      namePrefix: (path) => (path.includes('/blocks/') ? 'builder' : undefined),
-      cssNamespace: getSeededId,
+    vue2: vueConfig,
+    vue3: vueConfig,
+    reactNative: {
+      plugins: [
+        () => ({
+          code: {
+            pre: (code) => {
+              // workaround until we resolve
+              // https://github.com/BuilderIO/mitosis/issues/526
+              code.replace('srcset=', 'srcSet=');
```

Comment:
hacker status

Reply:
Those Mitosis maintainers take too long to respond to me and fix their bugs 😠